.env.example

```
@@ -1,5 +1,4 @@
-﻿# Database Configuration
-DATABASE_URL=postgresql://user:password@host/database?sslmode=require
-
-# Server Configuration
+﻿DATABASE_URL=postgresql://user:password@host/database?sslmode=require
 PORT=3000
+BETTER_AUTH_SECRET=H3QIYctDVe8jg2Ztk6K3xbPdfjvJkTQY
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BETTER_AUTH_SECRET key should go before the PORT key

(UnorderedKey)